azure-pipelines.yml

```diff
@@ -34,10 +30,16 @@ stages:
       displayName: Test PixelOven packages
     - script: yarn pixeloven:build:production
       displayName: Build PixelOven apps
-      # Need to trigger this only on master
     - script: yarn pixeloven:report:coverage
       displayName: Report PixelOven code coverage
-
-  # Need to do the same thing but for the examples... 
-  # Then we want to save coverage as an artifact and report coverage 
-  # yarn << parameters.type >>:report:coverage
\ No newline at end of file
+    # Run against examples now -- eventually this should be setup as a true integration step
+    - script: yarn examples:compile
+      displayName: Compile example packages
+    - script: yarn examples:lint
+      displayName: Lint example packages
+    - script: yarn examples:test
+      displayName: Test example packages
+    - script: yarn examples:build:production
+      displayName: Build example apps
+    - script: yarn examples:report:coverage
+      displayName: Report example code coverage
```

Comment:
Added the examples into the build. Also decided that it is ok we are reporting coverage stats for all branches. I think it could be helpful for referencing.